account_cost_center/__init__.py

```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from . import models
```

Comment:
Tested, works fine, but when you install it you need to update the account module to recreate the account.invoice.report database view. You could add a post_init_hook

```suggestion
from . import models
from odoo import api, SUPERUSER_ID
def recreate_invoice_report_view(cr, registry):
    env = api.Environment(cr, SUPERUSER_ID, {})
    env['account.invoice.report'].init()
```